lib/Recur/RRuleIterator.php

```diff
@@ -620,19 +620,33 @@ protected function nextYearly() {
 
         }
 
-        $currentMonth = $this->currentDate->format('n');
-        $currentYear = $this->currentDate->format('Y');
-        $currentDayOfMonth = $this->currentDate->format('j');
+        //$currentMonth = $this->currentDate->format('n');
+        //$currentYear = $this->currentDate->format('Y');
+        //$currentDayOfMonth = $this->currentDate->format('j');
```

Comment:
IF L623-L625 are no longer needed, they should be removed before merging.

Reply:
Gotcha